src/components/screens/wallet/overview/accountInfo/index.js

```diff
@@ -19,9 +19,92 @@ const BookmarkIcon = ({ bookmark }) => (
   />
 );
 
-/* eslint-disable complexity */
+const MultisigButton = ({ isMultisig, t }) => (
+  <div className={styles.helperIcon}>
+    <Tooltip
+      className={`${styles.tooltipWrapper} ${styles.centerContent} ${isMultisig ? styles.whiteBackground : ''}`}
+      position="bottom"
+      size="maxContent"
+      content={(
+        <DialogLink component={isMultisig ? 'multisigAccountDetails' : 'multiSignature'}>
+          <Icon name="multiSignature" />
+        </DialogLink>
+        )}
+    >
+      <p>{isMultisig ? t('View multisignature account details') : t('Register multisignature')}</p>
+    </Tooltip>
+  </div>
+);
+
+const BookmarkButton = ({
+  delegate, address, bookmark, host, t,
+}) => {
+  if (host === address) return null;
```

Comment:
Should this kind of validation not be done by the parent? If we do it like this, it is hiding some implementation detail that the component is not concerned with and should really exist in the parent. Also, in case when `host === address`, we will still see `BookmarkButton` in the react tree, albeit empty.

Reply:
It was to reduce the number of statements in the parent so we don't have to disable the eslint rule. but you're right in general.

Reply:
Done